1_The_Physical/Guess_Point_Level_1_Stateless/2_Arrange_Act_Assert/2_2_Stats_Calculator/src/index.ts

```diff
@@ -0,0 +1,36 @@
+type statistics = {
+  minimum: number;
+  maximum: number;
+  count: number;
+  average: number;
+};
+export class StatsCalculator {
+  public static calculateStats = (input: number[]): statistics => {
+    let minimum: number = input[0];
+    for (let index = 0; index < input.length; index++) {
+      minimum = input[index] < minimum ? input[index] : minimum;
+    }
```

Comment:
I would just move this piece of code into a function to be easier to read and make the main method even smaller. Same to fthe loops bellows